CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+From opening a bug report to creating a pull request: every contribution is
+appreciated and welcomed. If you're planning a new feature or change
+the api please create an issue first. This way we can ensure that your precious
+work is not in vain.
+
+## Issues
+
+Most of the time, when webpack does not work correctly, it might be a configuration issue.
+
+If you are still having difficulty after looking over your configuration carefully, please post
+a question to [StackOverflow with the webpack-cli tag](http://stackoverflow.com/tags/webpack-cli). Questions
+that include your webpack.config.js and relevant files, this way you help others to help you.
```

Comment:
Hi, I don't understand quite right what you mean here, would you be more specific about the change ?